dags/taar_amodump.py

```diff
@@ -1,83 +1,127 @@
-from airflow import DAG
 from datetime import datetime, timedelta
 
-from operators.emr_spark_operator import EMRSparkOperator
+from airflow import DAG
+from airflow.contrib.hooks.aws_hook import AwsHook
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.operators.subdag_operator import SubDagOperator
+
+from operators.gcp_container_operator import GKEPodOperator  # noqa
+from utils.dataproc import moz_dataproc_pyspark_runner
+
+cluster_name = "bq-load-gke-1"
```

Comment:
I would rename this variable to something like gke_cluster_name to avoid confusion. Then use the variable on lines 44, 64, 86